library/packages/src/lib/y2packager/release_notes_fetchers/rpm.rb

```diff
@@ -174,9 +174,15 @@ def release_notes_file(directory, prefs)
         langs << prefs.user_lang.split("_", 2).first if prefs.user_lang.include?("_")
         langs << prefs.fallback_lang
 
-        path = Dir.glob(
-          File.join(directory, "**", "RELEASE-NOTES.{#{langs.join(",")}}.#{prefs.format}")
-        ).first
+        path = nil
+        langs.any? do |lang|
```

Comment:
it is strange to call any when you ignore return value. So maybe each with break would be better?

Reply:
Hm, I was thinking how to implement this, I'd need something similar to `.find` but which return the  evaluated value, not the item from the list. `each` with `break` also does not look nice, but it's probably still a bit better...